index.js

```diff
@@ -13,12 +13,136 @@ mongoose
   .then(x => {
     console.log(`Connected to the database: "${x.connection.name}"`);
     // Before adding any recipes to the database, let's remove all existing ones
-    return Recipe.deleteMany()
+    return Recipe.deleteMany();
   })
-  .then(() => {
+  console.log(Recipe.title)
+  .then(() =>  {
+  //{
     // Run your code here, after you have insured that the connection was made
-    return Recipe.find({}, )// 
+    // return Recipe.create(
+    //   {
+    //     "title": "Chocolate Chip Cookies",
+    //     "level": "Amateur Chef",
+    //     "ingredients": [
+    //       "1/2 cup light brown sugar",
+    //       "1 large egg",
+    //       "2 tablespoons milk",
+    //       "1 1/4 teaspoons vanilla extract",
+    //       "2 cups semisweet chocolate chips"
+    //     ],
+    //     "cuisine": "French",
+    //     "dishType": "dessert",
+    //     "image": "https://imagesvc.meredithcorp.io/v3/mm/image?url=https%3A%2F%2Fimages.media-allrecipes.com%2Fuserphotos%2F4398987.jpg&w=596&h=399.32000000000005&c=sc&poi=face&q=85",
+    //     "duration": 30,
+    //     "creator": "Chef Jennifer"
+   //   });
+    return Recipe.insertMany([
+      {
+        "title": "Asian Glazed Chicken Thighs",
+        "level": "Amateur Chef",
+        "ingredients": [
+          "1/2 cup rice vinegar",
+          "5 tablespoons honey",
```

Comment:
At the top of this file we are importing all the data, so you can simply pass that array directly

`const data = require('./data');`

`Recipe.insertMany(data)`